ports/psoc6/mpconfigport.h

```diff
@@ -3,7 +3,7 @@
  *
  * The MIT License (MIT)
  *
- * Copyright (c) 2020-2021 Damien P. George
+ * Copyright (c) 2022-2023 Infineon Technologies AG
```

Comment:
I think we should keep the former copyright and add our own

Reply:
Well, our file bears no resemblance to the rp2 / minimal port files, so copyright should be IFX. Why should it be D. George ?

Reply:
I think we have to review ALL files before providing the port officially, because they all should have a common header / disclaimer as in the other ports. And many of our files do not.

Reply:
as it was based on the template and went over several iterations of changes... i guess it was an initial file with a copyright which we modified, and always public. But I am not a lawyer, and I don´t think this would create any legal problem anyhow :)